server/src/main/java/org/spine3/server/storage/memory/InMemoryRecordStorage.java

```diff
@@ -82,18 +82,16 @@ protected InMemoryRecordStorage(boolean multitenant) {
                         typeUrl = TypeUrl.of(state.getTypeUrl());
                     }
 
-                    final Any processed = AnyPacker.pack(
-                            FieldMasks.applyIfValid(
-                                    fieldMask,
-                                    AnyPacker.unpack(state),
-                                    typeUrl));
+                    final Message wholeState = AnyPacker.unpack(state);
+                    final Message maskedState = FieldMasks.applyIfValid(fieldMask, wholeState, typeUrl);
+                    final Any processed = AnyPacker.pack(maskedState);
 
                     matchingRecord.setState(processed);
 
                     result.add(matchingRecord.build());
-                    continue;
+                } else {
+                    result.add(null);
                 }
-                result.add(null);
```

Comment:
@armiol 
Also, I've spotted this code and decided that this is not quite obvious. Is this ok? Did `continue`-based way had some special sense?